doc/topics/installation/windows.rst

```diff
@@ -37,42 +37,95 @@ exist for earlier Salt Minion versions.
 
 .. note::
 
-    The installation executable installs dependencies that the Salt minion
+    The installation executable installs all dependencies that the Salt minion
     requires.
 
 The 64bit and 32bit installers have been tested on Windows 8.1, Windows Server
 2012 R2, Windows 10, Windows Server 2016, and Windows Server 2019. 32bit
 installers have only been tested on 64bit systems. Please file a bug report on
 our GitHub repo if issues for other platforms are found.
 
-The installer will detect previous installations of Salt and ask if you would
-like to remove them. Clicking OK will remove the Salt binaries and related files
-but leave any existing config, cache, and PKI information.
-
 Salt Minion Installation
 ========================
 
+Prerequisites and Checks
+------------------------
+
+Before the installation starts the installer does a few checks for prerequisites
+and security issues.
+
+Existing Salt Installation check
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If Salt is already installed on the system the user will be prompted to remove
+the previous installation. Click ``OK`` to remove the Salt binaries and related
+files but leave any existing config, cache, and PKI information. Click
+``Cancel`` to abort the installation before making any modifications to the
+system.
+
+Visual C++ Redistributable check
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
 If the system is missing the appropriate version of the Visual C++
 Redistributable (vcredist) the user will be prompted to install it. Click ``OK``
 to install the vcredist. Click ``Cancel`` to abort the installation without
 making modifications to the system.
 
-If Salt is already installed on the system the user will be prompted to remove
-the previous installation. Click ``OK`` to uninstall Salt without removing the
-configuration, PKI information, or cached files. Click ``Cancel`` to abort the
-installation before making any modifications to the system.
-
-After the Welcome and the License Agreement, the installer asks for two bits of
-information to configure the minion; the master hostname and the minion name.
-The installer will update the minion config with these options.
-
-If the installer finds an existing minion config file, these fields will be
-populated with values from the existing config, but they will be grayed out.
-There will also be a checkbox to use the existing config. If you continue, the
-existing config will be used. If the checkbox is unchecked, default values are
-displayed and can be changed. If you continue, the existing config file in
-``c:\salt\conf`` will be removed along with the ``c:\salt\conf\minion.d``
-directory. The values entered will be used with the default config.
+Insecure configuration check
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Since existing configuration that may be present on the system can be used by
```

Comment:
Is this only true of the 3004 installer? If so, we should note that, so one does not expect the older installers to behave this way.

Reply:
All our current installers behave this way.